WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -179,6 +177,55 @@ struct CouponDetails: View {
         .wooNavigationBarStyle()
     }
 
+    @ViewBuilder
+    private var amountTitleView: some View {
```

Comment:
❔ A general question about SwiftUI: how do we determine whether to create a separate SwiftUI view for a subview vs. defining a private view using `@ViewBuilder`? Just wondering why this view doesn't have its own view class, since it has its own constants and might be nice to have previews for itself

Reply:
Personally, I prefer keeping the subview as a variable and only extracting it to another view if it's either too big or reusable. In this case, I think the view is pretty small and it needs access to the parent view's state and view model, so keeping it in a variable is simpler.

Reply:
> might be nice to have previews for itself

This I agree, it may be more complicated to update the parent view's view model to test the subview in different states.